xapian-applications/omega/worker.cc

```diff
@@ -195,3 +194,9 @@ Worker::extract(const std::string& filename,
 
     return false;
 }
+
+string
+Worker::get_error()
+{
+    return error;
+}
```

Comment:
This should really be a `const` method, since it just inspects state and doesn't change the object it's called on.

Also, better to define a trivial getter like this inline in the class definition - that way the compiler can inline the method call and optimise it to just a string copy in the calling code.